src/index.js

```diff
@@ -258,7 +271,7 @@ async function parseMMD (browser, definition, outputFormat, { viewport, backgrou
  * @param {boolean} [opts.quiet] - If set, suppress log output.
```

Comment:
Assuming you move the default `outputFormat` logic into this `run()` function, we can use this doc string.

```suggestion
 * @param {boolean} [opts.quiet] - If set, suppress log output.
 * @param {"svg" | "png" | "pdf"} [opts.outputFormat] - Mermaid output format.
 * Defaults to `output` extension. Overrides `output` extension if set.
```

Reply:
Fixed in https://github.com/mermaid-js/mermaid-cli/pull/382/commits/fa79772dedda70470830ad2af17b0e25b5c89a7a